### ISSUE TITLE
All language: Missing i18n keys in yml files

### ISSUE THREAD

**Issue by kubedan** [telerik/kendo-angular-messages#108]
Missing a lot of i18n keys in yml files.

Missing all keys for:
kendo.chat
kendo.multiviewcalendar
kendo.editor
kendo.timeselector

**Comment by bernatgy** [comment]
A lot more is missing nowadays...

**Comment by Miedziaq** [comment]
yes missing, changed to meaning nothing, or just in english, but before update from 1.21 it was fine...

**Comment by dtopalov** [comment]
By default all locale-specific files and new messages are created in English, and all further translations for various languages are considered a community effort - https://github.com/telerik/kendo-angular-messages/blob/develop/CONTRIBUTING.md

If you have suggestions that would improve our messages in any language, PRs are most welcome. Thank you in advance.
